dataland-frontend/src/components/resources/frameworkDataSearch/ThreeLayerDataTable.vue

```diff
@@ -0,0 +1,326 @@
+<template>
+  <div v-show="mapOfCategoryKeysToDataObjectArrays.size > 0">
+    <DataTable tableClass="onlyHeaders">
```

Comment:
For Revenue, Operating Cost and Capital Assets please use the same formatting as is used in LkSG:

![image](https://github.com/d-fine/Dataland/assets/35459207/d55f47e9-7191-4411-9ee4-8fcc8084a7f6)

Reply:
Yes, that sounds reasonable.

Reply:
Done